.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: Main
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  all:
+    runs-on: windows-latest
+    timeout-minutes: 10
+
+    steps:
+
+      - uses: Brightspace/third-party-actions@actions/checkout
+      - uses: Brightspace/third-party-actions@actions/setup-dotnet
+        with:
+          dotnet-version: 7.0.x
+      - uses: Brightspace/third-party-actions@NuGet/setup-nuget 
+ 
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.1
+
+      - name: Install dependencies
+        run: nuget restore D2L.Hypermedia.Siren
+
+      - name: Build
+        run: msbuild -p:Configuration=Release D2L.Hypermedia.Siren

```

Comment:
I would rather not use `msbuild`, but nothing else works.  All combinations of `dotnet restore`, `dotnet build` and `dotnet msbuild` fail with this error.

<img width="1049" alt="image" src="https://github.com/Brightspace/D2L.Hypermedia.Siren/assets/34746763/36df3e2e-d3c0-4eb4-9b85-f4f36af482ed">

Reply:
Looks like the `.csproj` was created in VS and not with `dotnet new`, so the format is slightly different and `dotnet restore` can't find packages. Also, `dotnet restore` does not support `packages.config` that is used in this repo.

I could not make the build work with the current master, but when I switched to the commit prior to [this](https://github.com/Brightspace/D2L.Hypermedia.Siren/pull/36/files) change, I was able to build successfully with 
```
nuget restore
dotnet build -c Release
```

Looks like dbot updated `dotnet restore` but the [.csproj](https://github.com/Brightspace/D2L.Hypermedia.Siren/blob/c61262a22c5bd8f7c04e452f55fbec7019a6c149/D2L.Hypermedia.Siren/D2L.Hypermedia.Siren.csproj#L35) is still pointing to the old version.